filter_plugins/is_ipv6.py

```diff
@@ -0,0 +1,39 @@
+"""
```

Comment:
I think you can replace this with the built-in [ipv6 filter](http://docs.ansible.com/ansible/playbooks_filters_ipaddr.html)